core/templates.yml

```diff
@@ -25,595 +25,428 @@
 # The keys section contains the definitions for all the different keys that are being
 # used by Sgtk. A key is a magic token that is replaced by a value at runtime, for example
 # {Shot}. The section below tells Sgtk which data types and formats to expect for each key.
-#
+
```

Comment:
Something is very wrong here. Where did all of the path templates go? Seems like you have overwritten the existing defcon2 templates with just the ones from our gplstudio config. We need to make sure we preserve all of the existing templates, keys, and template strings from `tk-config-default2` while adding the ones we have from `tk-config-gplstudio`. Please fix this.

Reply:
oops you're right, I don't know what I did there. Will fix it.